doc/nrf/libraries/bin/lwm2m_carrier/app_integration.rst

```diff
@@ -145,39 +146,50 @@ Following are some of the general Kconfig options that you can configure:
 Server options
 ==============
 
-Following are some of the server Kconfig options that you can configure:
+Following are some of the server Kconfig options that you can configure.
+See the ``Enabled Carriers`` under :ref:`general_options_lwm2m` for when the option is relevant.
 
-The server settings can put the LwM2M carrier library either in the normal mode where it connects to the applicable carriers, or in the generic mode where it can connect to any server.
-
-* :kconfig:option:`CONFIG_LWM2M_CARRIER_IS_BOOTSTRAP_SERVER`:
-
-  * This configuration specifies if the :kconfig:option:`CONFIG_LWM2M_CARRIER_CUSTOM_URI` is an LwM2M Bootstrap Server.
+For :kconfig:option:`CONFIG_LWM2M_CARRIER_GENERIC`, no valid factory configuration has been set.
+At a minimum a URI must be set, unless the :kconfig:option:`CONFIG_LWM2M_SERVER_BINDING_CHOICE` is non-IP.
```

Comment:
We can add a remark that changing any of the server setting will result in a factory reset (new bootstrap sequence).